File: src/main/java/no/nav/syfo/services/exceptions/FastlegeIkkeFunnet.java
package no.nav.syfo.services.exceptions;

public class FastlegeIkkeFunnet extends RuntimeException {

    public static final String FASTLEGEIKKEFUNNET_MSG_DEFAULT = "Fant ikke aktiv fastlege";

    public FastlegeIkkeFunnet() {
        super(FASTLEGEIKKEFUNNET_MSG_DEFAULT);
    }

    public FastlegeIkkeFunnet(String message) {
        super(message);
    }
}

File: src/main/kotlin/no/nav/syfo/fastlege/exception/FastlegeIkkeFunnet.kt
package no.nav.syfo.fastlege.exception

class FastlegeIkkeFunnet : RuntimeException(FASTLEGEIKKEFUNNET_MSG_DEFAULT) {
    companion object {
        const val FASTLEGEIKKEFUNNET_MSG_DEFAULT = "Fant ikke aktiv fastlege"
    }
}
